frontend/benefit/applicant/src/components/applications/pageContent/PageContent.tsx

```diff
@@ -77,7 +77,7 @@ const PageContent: React.FC = () => {
       {currentStep === 1 && <ApplicationFormStep1 data={application} />}
       {currentStep === 2 && <ApplicationFormStep2 data={application} />}
       {currentStep === 3 && <ApplicationFormStep3 data={application} />}
-      {currentStep === 4 && <ApplicationFormStep4 />}
+      {currentStep === 4 && <ApplicationFormStep4 data={application} />}
```

Comment:
shared `ApplicationWizard` component is now in develop, so if you start to move to use it, only
```
<ApplicationWizard>
  <ApplicationFormStep1>
  <ApplicationFormStep2>
  <ApplicationFormStep3>
  ....
</ApplicationWizard>
```
is needed

Reply:
we will check it, but answered above. when we didnt have actual logic inside steps, our wizard implementation was the same simple and nice as yours actually. but then spex came:)